docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -138,11 +137,409 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=include-segment-file-sizes]
 [[cluster-nodes-stats-api-response-body]]
 ==== {api-response-body-title}
 
+[[cluster-nodes-stats-api-response-body-indices]]
+===== `indices` section
+
+`indices.docs.count`::
+(integer)
+include::{docdir}/rest-api/common-parms.asciidoc[tag=docs-count]
+
+`indices.docs.deleted`::
+(integer)
+include::{docdir}/rest-api/common-parms.asciidoc[tag=docs-deleted]
+
+`indices.store.size_in_bytes`::
+(integer)
```

Comment:
Do we differentiate between (Java) integer and (Java) long in the doc ? ...or is the just meant as whole number vs. fractional number thing ?

Reply:
It's meant as "whole number (integer) vs decimal number (float)".

That's consistent with how we've documented API reference so far, but I'm happy to make an exception here if needed. I could do something like `(64 bit signed integer)`.

I'd prefer not to refer folks to Java types if that's not strictly needed.